paddle/fluid/operators/conv_mkldnn_op.cc

```diff
@@ -298,6 +753,34 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
     return conv_attr;
   }
 
+  std::unique_ptr<mkldnn::convolution_forward::primitive_desc>
+  ConvFwdPrimitiveDesc(const memory::desc& src, const memory::desc& weights,
```

Comment:
This and the following ConvFwdPrimitiveDesc function (line 808) look very similar to it's fp32 counterparts. Couldn't we benefit here from the higher code re-use?